.travis.yml

```diff
@@ -20,41 +14,36 @@ language: python
 matrix:
   fast_finish: true
   allow_failures:
-    - env: REQUIREMENTS=devel EXTRAS=all,elasticsearch7 ES_URL=$ES7_DOWNLOAD_URL
+    - env: REQUIREMENTS=devel EXTRAS=all,elasticsearch7 ES_VERSION=ES_7_LATEST
     - python: 3.8
 
 cache:
   - pip
 
 services:
-  - postgresql
-  - redis
+  - docker
 
 env:
   global:
-    - ES5_DOWNLOAD_URL="https://artifacts.elastic.co/downloads/elasticsearch/elasticsearch-5.6.16.tar.gz"
-    - ES6_DOWNLOAD_URL="https://artifacts.elastic.co/downloads/elasticsearch/elasticsearch-6.8.2.tar.gz"
-    - ES7_DOWNLOAD_URL="https://artifacts.elastic.co/downloads/elasticsearch/elasticsearch-7.3.0-linux-x86_64.tar.gz"
-    - ES_HOST=127.0.0.1
-    - SQLALCHEMY_DATABASE_URI="postgresql+psycopg2://postgres@localhost:5432/invenio"
+    - DOCKER_COMPOSE_VERSION=1.26.2
   matrix:
-    - REQUIREMENTS=lowest EXTRAS=all,elasticsearch5 ES_URL=$ES5_DOWNLOAD_URL
-    - REQUIREMENTS=lowest EXTRAS=all,elasticsearch6 ES_URL=$ES6_DOWNLOAD_URL
-    - REQUIREMENTS=release EXTRAS=all,elasticsearch6 ES_URL=$ES6_DOWNLOAD_URL
-    - REQUIREMENTS=release EXTRAS=all,elasticsearch7 ES_URL=$ES7_DOWNLOAD_URL DEPLOY=true
-    - REQUIREMENTS=devel EXTRAS=all,elasticsearch7 ES_URL=$ES7_DOWNLOAD_URL
-
+    - REQUIREMENTS=lowest EXTRAS=all,elasticsearch5 ES_VERSION=ES_5_LATEST
+    - REQUIREMENTS=lowest EXTRAS=all,elasticsearch6 ES_VERSION=ES_6_LATEST
+    - REQUIREMENTS=release EXTRAS=all,elasticsearch6 ES_VERSION=ES_6_LATEST
+    - REQUIREMENTS=release EXTRAS=all,elasticsearch7 ES_VERSION=ES_7_LATEST
+    - REQUIREMENTS=devel EXTRAS=all,elasticsearch7 ES_VERSION=ES_7_LATEST
 
 python:
   - "3.6"
   - "3.7"
   - "3.8"
 
 before_install:
+  - sudo rm /usr/local/bin/docker-compose
```

Comment:
Comment: Could these lines be reduced to a oneliner via e.g. automation with docker-services-cli?

Reply:
this is needed for the travis vm... we could potentially run this after the install step (line 56). However, I don't think this pakcage should change "binaries of the computer". IMO it falls in the same category than `pre-requisites` in RDM... *If you want to use this, you need this that and that other min versions*